src/scss/_main.scss

```
@@ -581,16 +582,21 @@ nav {
 }
 
 .build-history {
-  display: flex;
-  flex-direction: row;
-  align-items: center;
-
-  &.-no-pad {
+  ul {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
     margin: 0;
     padding: 0;
   }
 }
 
+.build-history-title {
+  font-size: 80%;
```

Would you mind changing 80% to 0.8em?
Effectively the same but we're trying to be consistent. (when we can, and here, we can!)
```suggestion
  font-size: 0.8em;
```